go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	cloud.google.com/go v0.41.1-0.20190709211438-47e9997a900f
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/census-instrumentation/opencensus-proto v0.2.1
+	github.com/envoyproxy/envoy v1.13.1 // indirect
```

Comment:
I don't think this is needed. Envoy repo does not have any golang protos, `go-control-plane` does?